.vscode/launch.json

```diff
@@ -0,0 +1,33 @@
+{
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "name": "Launch via Node (en)",
+      "request": "launch",
+      "type": "node",
+      "program": "${workspaceFolder}/src/index.ts",
+      "runtimeVersion": "16",
+      "preLaunchTask": "tsc: build - tsconfig.json",
+      "skipFiles": [
+        "<node_internals>/**"
+      ],
+      "outFiles": [
+        "${workspaceFolder}/dist/**/*.js",
+      ]
+    },
+    {
+      "name": "Launch via Node (zh-tw)",
+      "request": "launch",
+      "type": "node",
+      "program": "${workspaceFolder}/src/index.ts",
+      "runtimeVersion": "16",
+      "preLaunchTask": "tsc: 建置 - tsconfig.json",
+      "skipFiles": [
+        "<node_internals>/**"
+      ],
+      "outFiles": [
+        "${workspaceFolder}/dist/**/*.js",
+      ]
+    }
+  ]
+}
```

Comment:
Is this file needed?

Reply:
It could be stashed locally, but if someone would like to use debugger to develop someday, the provided `launch.json` would be handy. Could be removed if redundant.